Octokit/Models/Response/ApiError.cs

```diff
@@ -9,6 +11,7 @@ namespace Octokit
 #if !NETFX_CORE
     [Serializable]
 #endif
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
```

Comment:
Since I added the [Error Property](https://github.com/octokit/octokit.net/pull/1061/files#diff-4628e8cb6fd5b6de8bb7d07ecc6a1e2bR38) the conventon test picked it up